ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-traffic-engineering.yml

```diff
@@ -0,0 +1,51 @@
+### Routing - Traffic Engineering ###
+router_traffic_engineering:
+  router_id:
+    ipv4: 10.0.0.1
+    ipv6: 2001:beef:cafe::1
+  segment_routing:
+    colored_tunnel_rib: true
+    policy_endpoint:
```

Comment:
Please make it policy_endpoints: ... and make it a list of dicts instead, same for colors. Example:

```yaml
...
    policy_endpoints:
      - address: < ipv4 or ipv6 address >
        colors:
          - value: < color value >
            binding_sid: < binding-sid >
            ...
```

This will also make your templating cleaner as you dont have to give a super-long path every time.